cmd/root.go

```diff
@@ -114,12 +111,8 @@ func init() {
 
 	rootCmd.SetVersionTemplate("{{with .Name}}{{printf \"%s \" .}}{{end}}{{printf \"v%s\" .Version}}\n")
 
-	l := logger.GetLogger()
-
 	// initialize all subcommands
 	rootCmd.AddCommand(observe.New())
-	rootCmd.AddCommand(serve.New(l))
-	rootCmd.AddCommand(status.New())
```

Comment:
I think there can still be a case to me made for the status command to stick around. It is only accessing the gRPC API, so it works nicely as a connectivity check from client to server. I see this being useful to app developers who want to troubleshoot their K8s application. 

The Hubble section to be added in `cilium status` on the other side I see more useful for infra admins deploying Cilium/Hubble. It also uses a different method to connect to the server: `cilium status` is using the local cilium unix socket only available on the node running cilium-agent, while `hubble status` would be intented to work from a client laptop.

Reply:
Makes sense. We can keep it.